tutorenrollmentreports/plugin.py

```diff
@@ -34,14 +35,14 @@
 }
 
 
-@local_command_group.command(help="Run the enrollment reports scrio")
+@local_command_group.command(help="Run the enrollment reports script")
 @click.pass_obj
 def enrollmentreports(context):
     config = tutor_config.load(context.root)
     job_runner = context.job_runner(config)
     job_runner.run_job(
         service="enrollmentreports",
-        command=f"bash -e run_enrollment_reports.sh"
+        command="ansible-playbook enrollment-report.yml"
```

Comment:
I am unable to understand this bit, if we don't use the bash script to start the `postfix` service and update the configurations. How are we going to take care of postfix service iside the container?

Reply:
Whichever way we do, it definitely won't be in such a way that you bake a script into the Docker image that contains the Postfix configuration.